.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -36,6 +36,7 @@
 -   [ ] If have added a new web framework, update the `add-ts-no-check.js` file to include that
```

Comment:
add changelog. In that, also mention that the way we check for sign up condition in functions override (like in email password sign up), has changed, and that you should also check for loginMethods length. Show code snippet too